### PR TITLE
Grails 6.2.0 upgrade

### PR DESCRIPTION
**Pull request by RxL-Farhan-Ali**

### CODE REVIEW

File: settings.gradle [hunk 1]
```diff
@@ -1,3 +1,33 @@
...
+DirectoryScanner.removeDefaultExclude('**/.hgsub')
+DirectoryScanner.removeDefaultExclude('**/.hgsubstate')
+DirectoryScanner.removeDefaultExclude('**/.hgtags')
+DirectoryScanner.removeDefaultExclude('**/.svn')
+DirectoryScanner.removeDefaultExclude('**/.svn/**')
+DirectoryScanner.removeDefaultExclude('**/CVS')
+DirectoryScanner.removeDefaultExclude('**/CVS/**')
+DirectoryScanner.removeDefaultExclude('**/SCCS')
+DirectoryScanner.removeDefaultExclude('**/SCCS/**')
+DirectoryScanner.removeDefaultExclude('**/vssver.scc')
```
Comment (parveen-rx) [inline settings.gradle]: any reason for this change?

File: gradle.properties [hunk 2]
```diff
@@ -1,4 +1,4 @@
-projectVersion=3.0.2-SNAPSHOT
-grailsVersion=5.3.2
-grailsGradlePluginVersion=5.3.0
-springSecurityCoreVersion=5.3.0
+projectVersion=4.0.0
```
Comment (parveen-rx) [inline gradle.properties:1]: Add below options too

org.gradle.caching=true
org.gradle.daemon=true
org.gradle.parallel=true
org.gradle.jvmargs=-Dfile.encoding=UTF-8 -Xmx1024M

File: build.gradle [hunk 3]
```diff
@@ -1,24 +1,15 @@
...
-}
-
-plugins {
-    id 'idea'
-    id 'io.github.gradle-nexus.publish-plugin' version '1.3.0'
-}
-
 allprojects {
-    apply plugin:"idea"
+    apply plugin: 'idea'
```
Comment (parveen-rx) [inline build.gradle:2]: Add below task for this project

1. bootrun block
2. sourceJar
3. packageJavadoc
4. GroovyCompile
5. jar
6. Test

**Review by parveen-rx — CHANGES_REQUESTED** [review]
@RxL-Farhan-Ali : Please also create changelog.md file similar to other plugin

File: settings.gradle (hunk 1 above)
Comment (RxL-Farhan-Ali) [reply settings.gradle]: I was getting error while building the project and the only solution which worked is the above one for me.

File: build.gradle (hunk 3 above)
Comment (RxL-Farhan-Ali) [reply build.gradle:2]: bootRun task added and for sourceJar, packageJavaDoc,GroovyCompile already task utilized default one and test junit is already available.

File: gradle.properties (hunk 2 above)
Comment (RxL-Farhan-Ali) [reply gradle.properties:1]: Added, org.gradle.parallel=true giving error while building.

**Review by parveen-rx — APPROVED** [review]

**Review by parveen-rx — APPROVED** [review]

**Comment by parveen-rx** [comment]
@vsachinv : Merge it, please review when you get time, comments can be fixed in another PR if needed

**Pull request merged**